payu/models/cesm_cmeps.py

```diff
@@ -115,13 +115,20 @@ def set_model_pathnames(self):
 
         super().set_model_pathnames()
 
-        self.work_input_path = os.path.join(self.work_path, 'input')
+        self.work_input_path = os.path.join(self.work_path, 'INPUT')
         
+        # MOM restarts are dealt with via pointer files (see below). Use work_restart_path
+        # for additional restarts (e.g. generic tracer flux restarts)
+        self.get_runconfig(self.control_path)
+        additional_restart_dir = self.runconfig.get(
+            "ALLCOMP_attributes", "additional_restart_dir", "RESTART/"
```

Comment:
Standard practice in other model drivers (well `fms`) is not to include a trailing slash, and you've not used one for `INPUT`.

Reply:
IIRC, the trailing slash is _required_ by the FMS routines that use this... I'll look into this.

Reply:
Actually I fixed this. But I'm being dumb and it obviously doesn't matter here either way... Resolved in [6a10afd](https://github.com/payu-org/payu/pull/433/commits/6a10afd11512eec4d41fd5ad74833245b63b8d1b)